saas/control_app/src/justfile

```
@@ -0,0 +1,53 @@
+set fallback
+set dotenv-load := true
+set shell := ["bash", "-euxc", "-o", "pipefail"]
+
+default:
+    @{{just_executable()}} --list
+
+py-lint:
+  python3 -m pylint ./TenantManagement --rcfile .pylintrc
+
+py-test:
+  python3 -m pytest ./TenantManagement/tests/unit
+
+py-test-debug:
+  python3 -m pytest --trace --pdbcls=IPython.terminal.debugger:TerminalPdb ./TenantManagement/tests/unit
+
+py-install:
+  python3 -m pip install -r ./TenantManagement/requirements.txt
+
+dynamodb-create-table:
+  # TODO use JSON schema
+  AWS_ACCESS_KEY_ID=X AWS_SECRET_ACCESS_KEY=X aws dynamodb create-table --endpoint-url http://localhost:8000 \
+  --table-name saas \
+  --attribute-definitions AttributeName=dataType,AttributeType=S AttributeName=dataId,AttributeType=S \
+  --key-schema AttributeName=dataType,KeyType=HASH AttributeName=dataId,KeyType=RANGE \
+  --provisioned-throughput ReadCapacityUnits=10,WriteCapacityUnits=5 \
+  --region ddblocal
+
+dynamodb-get-item type id:
+  #!/bin/bash
+  AWS_ACCESS_KEY_ID=X AWS_SECRET_ACCESS_KEY=X aws dynamodb get-item \ 
+  --endpoint-url http://localhost:8000 \ 
+  --table-name saas \ 
+  --key '{"dataType": {"S": "{{type}}"}, "dataId": {"S":"{{id}}"}}' \ 
+  --region ddblocal
+
+sam-start-dynamodb:
+  #!/bin/bash
+  if ! docker network ls | grep lambda-local
+  then
+    docker network create lambda-local
+  fi
+  docker rm -f /dynamodb
+  docker run -p 8000:8000 --network lambda-local --name dynamodb amazon/dynamodb-local -jar DynamoDBLocal.jar -sharedDb -inMemory  
```

would it make sense to run this in detach mode?
```suggestion
  docker run -p 8000:8000 -d --network lambda-local --name dynamodb amazon/dynamodb-local -jar DynamoDBLocal.jar -sharedDb -inMemory  
```

---

I kind of prefare to stop those in place when I don't need them anymore. so I don't forget about it